Resources/public/js/views/fields/ez-relationlist-editview.js

```diff
@@ -50,7 +51,57 @@ YUI.add('ez-relationlist-editview', function (Y) {
                 } else {
                     this.render();
                 }
+
+                clearTimeout(this.makeListSortableTimeout);
+                this.makeListSortableTimeout = setTimeout(Y.bind(this._makeListSortable, this), 75);
+            });
+        },
+        
+        /**
+         * Sets up the relation list to be sortable
+         */
+        _makeListSortable: function() {
+            var fieldDefinitionIdentifier = this.getField().fieldDefinitionIdentifier;
+    
+            var sortable = new Y.Sortable({
+                container: "." + fieldDefinitionIdentifier + ".ez-relation-container",
+                nodes: ".ez-relation-content",
+                opacity: ".1"
+            });
+    
+            sortable.delegate.after("drag:end",Y.bind(function(e) {
+                this.__setSortedObjectRelationss(e);
```

Comment:
still seems to be typo here or? (sending with `ss` as opposed to `s`)

Reply:
Fixed in d930c91